modules/tensor_mechanics/test/tests/j_integral/gold/j_integral_2d_mouth_dir_out.csv

```diff
@@ -1,4 +1,3 @@
 time,J_1,J_2,J_3,J_4,J_5
 0,0,0,0,0,0
-1,1.1655970440759,1.1625839930871,1.1613679143305,1.164168518934,1.1632303584855
```

Comment:
Why don't we get exactly double on this file?

Reply:
I do not have an explanation. From the code in Jintegral.C
https://github.com/idaholab/moose/blob/51557c3cd1fd65ea9518d83a221f514524c39c99/modules/tensor_mechanics/src/vectorpostprocessors/JIntegral.C#L228-L229
the only place where the symmetry information takes effect is here, by doubling the j_integral result.
`modules/tensor_mechanics/test/tests/j_integral/gold/j_integral_3d_mouth_dir_out.csv` this one is the other result that does not give us exact doubling. They both use `crack_direction_method = CrackMouth` and default `crack_end_direction_method=NoSpecialTreatment`. The other tests with `crack_direction_method = CrackMouth` but `crack_end_direction_method=CrackDirectionVector` satisfy the double relationship.

Reply:
There is a piece of code that modifies the location of the crack front nodes here:

https://github.com/idaholab/moose/blob/51557c3cd1fd65ea9518d83a221f514524c39c99/modules/tensor_mechanics/src/userobjects/CrackFrontDefinition.C#L1063-L1066

I don't fully understand why we are zeroing the components of the symmetry plane for the entire crack mouth definition. It seems as if the nodes of the crack mouth could potentially move along the symmetry. If you change the symmetry plane to "2", you get exactly double the value (but it's obviously wrong in this case).

Reply:
@bwspenc: We can merge this PR once the tests pass, but I think we should address the reason why we don't get exactly double when we add the correct symmetry plane. I can open an issue if that cannot be address now.